test/checks/navigation/heading-order.js

```diff
@@ -117,8 +118,8 @@ describe('heading-order', function() {
     });
   });
 
-  it('should ignore aria-level on iframe when not used with role=heading', function() {
-    var vNode = queryFixture('<iframe aria-level="2"></iframe>');
+  it('should ignore aria-level on iframe when not used with role=heading', function () {
+    var vNode = fixtureSetup('<iframe aria-level="2"></iframe>');
```

Comment:
Even though `heading-order` doesn't make use of `vNode` anymore, for now we should be consistent, so adding a `#target` would be more appropriate rather than trying to save off `fixtureSetup` which returns the entire virtual node tree (which would break the test if we ever change how `heading-order` works in the future).

```suggestion
    var vNode = queryFixture('<iframe aria-level="2" id="target"></iframe>');
```

Reply:
Updated in 36bbd049b1a1c2d7ff517a44d0f74317191fabbf